install.py

```diff
@@ -391,6 +391,9 @@ def install(inst, version):
     else:
         oqreal = '%s/bin/oq' % inst.VENV
 
+    if inst in (user, devel):  # create the db in the default location
+        subprocess.check_call([oqreal, 'dbserver', 'createdb'])
+
```

Comment:
in the installation is server the db is created with the start of systemctl service?

Reply:
Yes.

Reply:
LGTM